CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fixed trivy security issue CVE-2024-1597
 - Fixed spring-security-core issue CVE-2024-22234
+- Changed image type in ARC42 to svg.
```

Comment:
This is not a change I would highlight in the `CHANGELOG`.

Reply:
sure @SebastianBezold . I have removed this line. thanks